WalletWasabi.Fluent/Global.cs

```diff
@@ -373,6 +378,11 @@ public async Task DisposeAsync()
 					await httpClientFactory.DisposeAsync().ConfigureAwait(false);
 					Logger.LogInfo($"{nameof(HttpClientFactory)} is disposed.");
 				}
+				if (CoordinatorHttpClientFactory is { } coordinatorHttpClientFactory)
```

Comment:
This pattern `if(someNullableThing is {} nonNullThing) { dispose }`  is here because in the past all those properties were nullable but that's not true anymore, in this specific case CoordinatorHttpClientFactory is never null i think.

If you remove this `if` statement then I think CodeScene will be happy.

Reply:
Gotcha, done